merlin/examples/workflows/feature_demo/feature_demo.yaml

```diff
@@ -56,22 +53,22 @@ study:
       description: |
          train a learner on the results
       run:
-         cmd: python3 $(LEARN) -infile $(translate.workspace)/results.npz
+         cmd: spellbook learn -infile $(translate.workspace)/results.npz -outfile regressor.pkl
```

Comment:
feature_demo needs its requirements.txt updated to include the spellbook

Reply:
Good catch